final_project.py

```
@@ -0,0 +1,196 @@
+Original file is located at
+    https://colab.research.google.com/drive/1KP92gjlVDWp_YKdKsk7M5Cp11H9UE4kv
+"""
+
+from google.colab import files
+uploaded = files.upload()
+
+import pandas as pd
+import numpy as np
+
+df = pd.read_csv('HR_Data.csv')
+df
```

Ruff (0.12.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)